spec/services/proofing/resolution/progressive_proofer_spec.rb

```diff
@@ -136,82 +188,53 @@
 
-          device_profiling_result = result.device_profiling_result
+        it 'returns a disabled result' do
+          device_profiling_result = proof.device_profiling_result
 
           expect(device_profiling_result.success).to be(true)
           expect(device_profiling_result.client).to eq('tmx_disabled')
           expect(device_profiling_result.review_status).to eq('pass')
         end
       end
 
+      # Remove the mocks:
+      #   `Proofing::LexisNexis::InstantVerify::Proofer#new`
+      #   `Proofing::LexisNexis::InstantVerify#new`
+      # in the outermost `before` block after removing this context.
       context 'LexisNexis Instant Verify A/B test enabled' do
-        let(:residential_instant_verify_proof) do
-          instance_double(Proofing::Resolution::Result)
-        end
-        let(:instant_verify_workflow) { 'equitable_workflow' }
         let(:ab_test_variables) do
           {
             ab_testing_enabled: true,
             use_alternate_workflow: true,
-            instant_verify_workflow: instant_verify_workflow,
+            instant_verify_workflow: 'equitable_workflow',
           }
         end
 
-        before do
-          allow(instant_verify_proofer).to receive(:proof).
-            and_return(residential_instant_verify_proof)
-          allow(residential_instant_verify_proof).to receive(:success?).and_return(true)
-        end
+        before { proof }
 
         it 'uses the selected workflow' do
-          lniv = Idv::LexisNexisInstantVerify.new(dcs_uuid)
-          expect(lniv).to receive(:workflow_ab_testing_variables).
-            and_return(ab_test_variables)
-          expect(Idv::LexisNexisInstantVerify).to receive(:new).
-            and_return(lniv)
-          expect(Proofing::LexisNexis::InstantVerify::Proofer).to receive(:new).
-            with(hash_including(instant_verify_workflow: instant_verify_workflow)).
-            and_return(instant_verify_proofer)
-
-          proof
+          expect(Proofing::LexisNexis::InstantVerify::Proofer).to(
+            have_received(:new).with(
+              hash_including(
+                instant_verify_workflow: 'equitable_workflow',
+              ),
+            ),
+          )
         end
       end
 
       context 'remote flow does not augment pii' do
-        let(:aamva_proofer) { instance_double(Proofing::Aamva::Proofer) }
-        let(:id_address_instant_verify_proof) do
-          instance_double(Proofing::Resolution::Result)
-        end
-
-        before do
-          allow(instance).to receive(:state_id_proofer).and_return(aamva_proofer)
-          allow(instance).to receive(:resolution_proofer).and_return(instant_verify_proofer)
-          allow(instant_verify_proofer).to receive(:proof).
-            and_return(id_address_instant_verify_proof)
-          allow(id_address_instant_verify_proof).to receive(:success?).and_return(true)
-        end
+        before { proof }
 
         it 'proofs with untransformed pii' do
```

Comment:
seems like this is part of the expectation of the test. it's easier to debug/override when important things happen int the spec instead of a `before`

```suggestion
        it 'proofs with untransformed pii' do
          proof

```